pkg/orchestration/wiring/postgres/plugin.go

```diff
@@ -229,40 +190,15 @@ func instanceParameters(resource *orch_v1.StateResource, context *wiringplugin.W
 	// Filter instanceId
 	delete(finalSpec, "instanceId")
 
-	if finalSpec == nil {
```

Comment:
This is never true because it is always initialized above to a non-nil value (unless there is an error in which case we exit earlier).